sdk/core/Azure.Core/src/Shared/ThrowHelper.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Copied from https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.Extensions.Primitives/src
+
+using System;
+using System.Diagnostics;
+
+#pragma warning disable CA1305  // ToString Locale
+#pragma warning disable IDE0051 // Private member not used
+
+namespace Azure.Core
+{
+    internal static class ThrowHelper
```

Comment:
This one may confuse people considering we have `Arguments` for parameter validation. Can we inline/trim/move into Multipart?

Reply:
Turns out it is not used anymore - deleted